lib/human.js

```diff
@@ -92,29 +102,36 @@ const formattedCorrection = (spec, result) => {
 
   if (_.isEmpty(suggested_corrections)) return null;
 
-  return '  You fu*ked up, here\'s how to fix it:\n\n'
+  return '  ' + errorMessage(custom).conclusion + '\n\n'
     + suggested_corrections.join('\n');
 };
 
 
-// readableErrorMessage :: ( Spec, PropCheckResult ) -> String
-const readableErrorMessage = _.curryN(2, _.compose(
-  _.join('\n\n')
+// wrapString :: String -> String -> String
+const wrapString = _.curryN(2, (wrapper, originalString) => {
+  return wrapper + originalString + wrapper;
+});
+
+
+// readableErrorMessage :: CustomMessage -> ( Spec, PropCheckResult ) -> String
+const readableErrorMessage = (custom) => _.curryN(2, _.compose(
+  wrapString('\n')
+, _.join('\n\n')
 , _.filter(notNullOrEmpty)
 , _.juxt([
-    formattedUserInput
-  , formattedIncorrectInput
-  , formattedExpectedInput
-  , formattedCorrection
+    formattedUserInput(custom)
```

Comment:
Perhaps this should just be a `_.curryN(3, ...` since all of the functions in the `juxt` call take the parameter.